src/AmplifyMapLibreRequest.ts

```diff
@@ -12,6 +12,7 @@ import {
   RequestParameters,
   MapOptions,
 } from "maplibre-gl";
+import extractDomain from "extract-domain";
```

Comment:
I don't think we should add another depedency to this package for this. You can use: https://developer.mozilla.org/en-US/docs/Web/API/URL/URL

Reply:
Removed dependency